files/zh-cn/web/html/element/b/index.md

```diff
@@ -1,68 +1,105 @@
-   <td><a href="/zh-CN/docs/HTML/Content_categories#Phrasing_content">Phrasing content</a>.</td>
-  </tr>
-  <tr>
-   <th scope="row">标签略写</th>
-   <td>{{no_tag_omission}}</td>
-  </tr>
-  <tr>
-   <th scope="row">允许的父元素</th>
-   <td>任意可容纳 <a href="/zh-CN/docs/HTML/Content_categories#Phrasing_content">phrasing content</a> 的元素</td>
-  </tr>
-  <tr>
-   <th scope="row">Permitted ARIA roles</th>
-   <td>任意</td>
-  </tr>
-  <tr>
-   <th scope="row">DOM 接口</th>
-   <td>{{domxref("HTMLElement")}}</td>
-  </tr>
- </tbody>
-</table>
+**`<b>`** 元素 [HTML](/zh-CN/docs/Web/HTML) 用于吸引读者注意元素内容，而这些内容本身并不具有特别重要性。它以前被称为粗体元素，大多数浏览器仍然以粗体显示文本。但是，你不应该使用 `<b>` 来设置文本样式或赋予重要性。如果你希望创建粗体文本，应该使用 CSS {{cssxref("font-weight")}} 属性。如果你希望指示某个元素特别重要，应该使用 {{HTMLElement("strong")}} 元素。
+
+{{EmbedInteractiveExample("pages/tabbed/b.html", "tabbed-shorter")}}
 
 ## 属性
 
-这个元素只包含[全局属性](/zh-CN/docs/HTML/Global_attributes)。
+这个元素只包含 [全局属性](/zh-CN/docs/HTML/Global_attributes)。
 
 ## 使用说明
 
-- `<b>` 应用场合如：摘要中的关键字、评论中的产品名称，或其他典型的应该加粗显示的文字（除此之外无其他特别强调）。
-- 不要将\<b>元素与 {{HTMLElement("strong")}}、{{HTMLElement("em")}}或{{HTMLElement("mark")}}元素混淆。 {{HTMLElement("strong")}}元素表示某些重要性的文本，{{HTMLElement("em")}}强调文本，而{{HTMLElement("mark")}}元素表示某些相关性的文本。 `<b>`元素不传达这样的特殊语义信息；仅在没有其他合适的元素时使用它。
-- 类似的，也不要用 `<b>` 元素来标记标题。如果需要表示标题，请使用 {{HTMLElement("h1")}} 到 {{HTMLElement("h6")}} 标签。此外，可以通过样式表来改变这些元素的默认样式，结果就是它们并*不一定*都是粗体。
-- 通过在 `<b>` 元素上添加 [`class`](/zh-CN/docs/Web/HTML/Global_attributes#class) 属性来表示额外的语义信息是一个很好的方式（例如 `<b class="lead">` 来表示段落中的第一句）。这可以在样式需要变动的情况下更轻松地处理不同应用场合的 `<b>` 元素，无须改动 HTML。
-- 以前 `<b>` 元素的意思就是让文本变成粗体。但从 HTML4 开始，不赞成标签表示带样式信息，于是 `<b>` 元素的含义发生了变化。
-- 如果不是出于语义目的而使用 `<b>` 元素，那么让文本显示粗体更好的方式是使用将 CSS 的 {{cssxref("font-weight")}} 属性设置为 `"bold"`。
+- 在以下情况下使用 `<b>` 元素：摘要中的关键字、评论中的产品名称，其他通常以粗体显示的文本（但不包括任何特别重要的内容）。
+- 不要将 `<b>` 元素与 {{HTMLElement("strong")}}、{{HTMLElement("em")}} 或 {{HTMLElement("mark")}} 元素混淆。{{HTMLElement("strong")}} 元素表示具有某种重要性的文本，{{HTMLElement("em")}} 元素强调文本，而 {{HTMLElement("mark")}} 元素表示具有某种关联性的文本。`<b>` 元素不会传达这种特殊的语义信息；仅在其他元素都不适用时使用它。
+- 同样，不要使用 `<b>` 元素标记标题和页眉。为此，请使用 {{HTMLElement("h1")}} 到 {{HTMLElement("h6")}} 标签。此外，样式表可以更改这些元素的默认样式，结果它们 _不一定_ 以粗体显示。
+- 好的做法是在 `<b>` 元素上添加 [`class`](/zh-CN/docs/Web/HTML/Global_attributes/class) 属性来传达额外的语义信息（例如，段落中的第一个句子可以使用 `<b class="lead">`）。这样，如果你的样式需求发生变化，可以更轻松地管理 `<b>` 的多个用例，从而无需更改 HTML。
+- 历史上， `<b>` 元素的目的是使文本加粗。自 HTML4 以来，样式信息已被弃用，因此 `<b>` 元素的含义已更改。
+- 如果使用 `<b>` 元素没有语义目的，你应该使用 CSS 的 {{cssxref("font-weight")}} 属性并将值设置为 `"bold"` 来使文本加粗。
 
-## 例子
+## 示例
 
 ```html
 <p>
-  This article describes several <b>text-level</b> elements. It explains their
-  usage in an <b>HTML</b> document.
+  本文档描述了几个 <b>文本级</b> 元素。它解释了它们在 <b>HTML</b> 文档中的用法。
 </p>
-Keywords are displayed with the default style of the
-<b> element, likely in bold.</b>
+关键字以 <b>元素的默认样式显示，可能是粗体。</b>
 ```
 
 ### 结果
 
-This article describes several **text-level** elements. It explains their usage in an **HTML** document.
+{{EmbedLiveSample("Examples")}}
```

Comment:
Please check [this section](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E7%BF%BB%E8%AF%91%E7%A4%BA%E4%BE%8B%E4%BB%A3%E7%A0%81%E5%9D%97)

```suggestion
{{EmbedLiveSample("示例")}}
```